project/plugins.sbt

```diff
@@ -0,0 +1,24 @@
+lazy val sbtCiReleaseVersion = "1.5.10"
+
+lazy val ow2Version = "9.5"
+lazy val jacocoVersion = "0.8.10-absa.1"
+val sbtJacocoVersion = "3.4.1-absa.3"
+val scalaArmVersion = "2.0"
+
+def jacocoUrl(artifactName: String): String = s"https://github.com/AbsaOSS/jacoco/releases/download/$jacocoVersion/org.jacoco.$artifactName-$jacocoVersion.jar"
+def ow2Url(artifactName: String): String = s"https://repo1.maven.org/maven2/org/ow2/asm/$artifactName/$ow2Version/$artifactName-$ow2Version.jar"
+def armUrl(scalaMajor: String): String = s"https://repo1.maven.org/maven2/com/jsuereth/scala-arm_$scalaMajor/$scalaArmVersion/scala-arm_$scalaMajor-$scalaArmVersion.jar"
+
+addSbtPlugin("com.github.sbt" % "sbt-ci-release" % sbtCiReleaseVersion)
+
+addSbtPlugin("com.jsuereth" %% "scala-arm" % scalaArmVersion from armUrl("2.11"))
+addSbtPlugin("com.jsuereth" %% "scala-arm" % scalaArmVersion from armUrl("2.12"))
```

Comment:
Why do we need these plugins?

Reply:
Originally added by Miroslav with Jacoco PR, and my PR was originally pointed to that, so I kept it. So the answer is that it seems to be related to jacoco.

But, I checked usages: http://jsuereth.com/scala-arm/usage.html and I don't think it's really used. @miroslavpojer what do you think?

Reply:
Actually without this plugin, the build fails; it seems to be a prerequisite for jacoco:

```
[error] java.lang.NoClassDefFoundError: resource/package$
[error] 	at com.github.sbt.jacoco.data.InstrumentationUtils$.$anonfun$instrumentClasses$7(InstrumentationUtils.scala:59)
[error] 	at com.github.sbt.jacoco.data.InstrumentationUtils$.$anonfun$instrumentClasses$7$adapted(InstrumentationUtils.scala:58)
[error] 	at scala.collection.immutable.List.foreach(List.scala:431)
[error] 	at com.github.sbt.jacoco.data.InstrumentationUtils$.instrumentClasses(InstrumentationUtils.scala:58)
[error] 	at com.github.sbt.jacoco.BaseJacocoPlugin.$anonfun$taskValues$9(BaseJacocoPlugin.scala:117)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:63)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:69)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:283)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:24)
[error] 	at sbt.Execute.work(Execute.scala:292)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:283)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:65)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] Caused by: java.lang.ClassNotFoundException: resource.package$
[error] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
[error] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
[error] 	at com.github.sbt.jacoco.data.InstrumentationUtils$.$anonfun$instrumentClasses$7(InstrumentationUtils.scala:59)
[error] 	at com.github.sbt.jacoco.data.InstrumentationUtils$.$anonfun$instrumentClasses$7$adapted(InstrumentationUtils.scala:58)
[error] 	at scala.collection.immutable.List.foreach(List.scala:431)
[error] 	at com.github.sbt.jacoco.data.InstrumentationUtils$.instrumentClasses(InstrumentationUtils.scala:58)
[error] 	at com.github.sbt.jacoco.BaseJacocoPlugin.$anonfun$taskValues$9(BaseJacocoPlugin.scala:117)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:63)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:69)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:283)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:24)
[error] 	at sbt.Execute.work(Execute.scala:292)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:283)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:65)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] (balta / Test / fullClasspath) java.lang.NoClassDefFoundError: resource/package$
[error] Total time: 10 s, completed Dec 11, 2023 12:46:13 PM
```

Reply:
Yes, it is needed. Whole this concept could be removed when I will find time to migrate scala improvement back to JaCoCo origin project. Sorry for chaos here.

Reply:
I could try to add these dependencies inside to our fork release when time capacities allow me it.